tests/testthat/test-use-data-table.R

```diff
@@ -0,0 +1,20 @@
+context("use_data_table")
```

Comment:
`context()` is no longer necessary and is mildly discouraged. So this should be deleted.

Reply:
I realize we still have some pre-existing `context()` calls, but we shouldn't be adding any new ones.